src/Stratis.BreezeD/Properties/launchSettings.json

```diff
@@ -14,7 +14,7 @@
     },
     "Stratis Test": {
       "commandName": "Project",
-      "commandLineArgs": "stratis -testnet -debug=all -loglevel=debug"
+      "commandLineArgs": "stratis -testnet -prune=2880"
```

Comment:
Maybe create a new one here.

Reply:
I changed this so that we always start up breeze here with pruning on ;)

How do you mean "create a new one"?